ClientGenerator/src/googleapis/codegen/languages/csharp/default/templates/___package_name___/app.netstandard10.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<configuration>
+  <runtime>
```

Comment:
Yes, all required due to PCL requirements.
Removing any of these produces warnings in the restore/build.